include/irr/core/math/aabb2diSIMD.h

```diff
@@ -0,0 +1,155 @@
+#ifndef __IRR_AABB_SIMD_2D_INT_H_INCLUDED__
+#define __IRR_AABB_SIMD_2D_INT_H_INCLUDED__
+
+#include "irr/core/math/irrMath.h"
+#include "irr/core/memory/memory.h"
+#include "irr/core/alloc/AlignedBase.h"
+
+namespace irr { namespace core {
+
+class aabb2dSIMDi
+{
+public:
+	inline aabb2dSIMDi()
+		: internalBoxRepresentation(-INT_MAX, -INT_MAX, -INT_MAX, -INT_MAX) {};
```

Comment:
just INT_MIN instead

Reply:
actually because INT_MIN is equal to -1-INT_MAX, we cannot use the floating point hack :(